src/docs/product/cli/dif.mdx

```diff
@@ -265,31 +265,32 @@ because ProGuard files work on projects. For more information about this refer t
 The `upload-proguard` command is the one to use for uploading ProGuard files. It
 takes the path to one or more ProGuard mapping files and will upload them to
 Sentry.
-Example:
 
 ```bash
 sentry-cli upload-proguard \
     app/build/outputs/mapping/{BuildVariant}/mapping.txt
 ```
 
-Since the Android SDK needs to know the UUID of the mapping file, you need
-to associate it with the upload. If you supply `--uuid`, it sets that value:
-
-```bash
-sentry-cli upload-proguard \
-    --uuid A_VALID_UUID \
-    app/build/outputs/mapping/{BuildVariant}/mapping.txt
-```
-
-Which then also needs to be included in your `AndroidManifest.xml` file:
+Since the Android Sentry SDK needs to know the UUID of the mapping file, you need
+to associate it with the upload. However, you first have to place that UUID to the `AndroidManifest.xml` file:
```

Comment:
```suggestion
Since the Android Sentry SDK needs to know the UUID of the mapping file, you need
to associate it with the upload. However, you first have to place that UUID into the `AndroidManifest.xml` file:
```